pandas/info.py

```diff
@@ -1,20 +0,0 @@
-"""
-pandas - a powerful data analysis and manipulation library for Python
-=====================================================================
-
-See http://pandas.pydata.org/ for full documentation. Otherwise, see the
```

Comment:
I could theorectically replace this with something (maybe a minimal README type of thing, so its on the actual doc-string). any thoughts?